src/arg.rs

```diff
@@ -33,6 +34,14 @@ pub fn parse_args() -> Args {
                 .value_parser(clap::value_parser!(u64))
                 .help("Minimum size to clear (in bytes)"),
         )
+        .arg(
+            Arg::new("dry_run")
+                .short('n')
```

Comment:
-n is not very intuitive for dry run.
Maybe -dr ? -r ?

Reply:
yeah I though so too, was gonna go for -dr but it needs to be a character. d wasn't an option cause it's used. r feels like recursive but maybe better.  Will change